gno.land/pkg/gnoclient/integration_test.go

```diff
@@ -113,6 +114,64 @@ func TestCallMultiple_Integration(t *testing.T) {
 	assert.Equal(t, expected, got)
 }
 
+func TestCallMultipleWithEvent_Integration(t *testing.T) {
```

Comment:
Could you test generating several events between different blocks and different transactions? thanks

Reply:
I'll add test that tests `multi tx + mutli msg`.

But I don't think testing events between different block is necessary test, different block really has nothing to do with `multi-msg`